site/versions/master/docs/install.md

```diff
@@ -216,6 +220,7 @@ $ export PATH="$PATH:$HOME/bin"
 
 You can also add this command to your `~/.bashrc` file.
 
+You are all set. You can confirm Bazel is install successfully by running `$ bazel`.
```

Comment:
s/install/installed/

Also, `bazel version` instead of `$ bazel` (don't include the $ for inline snippets).